docs/main/ios/spm.md

```diff
@@ -59,7 +59,45 @@ Then let's sync the web app. This will add the App plugin SPM to the iOS project
 
 You can now use the App plugin normally.
 
-<em>More details coming soon</em>
+## Using SPM in an existing Capacitor Project
+
+### Using our migration tool
+
+The Capacitor CLI has a command to help migrate from CocoaPods to Swift Package Manager. However, one manual step is still required. In addition, projects with Cordova plugins will not be migrated correctly and neither will projects that use plugins that do not have SPM versions available.
+
+To start, run `npx cap migrate-to-spm` in the root of your project.
+
+This tool will:
+  - Run `pod deintegrate` removing CocoaPods
+  - Delete the `Podfile`, `App.xcworkspace`, and `Podfile.lock`
+  - Create a `CapApp-SPM` directory with the needed files
+  - Generate a `Package.swift` from your plugins, and warn you if any can't included. 
+
+After this is run, run a `npx cap sync` again.
+
+Then run `npx cap open ios` or open `App.xcodeproj` and you should see something similar to this:
+
+![Migrate Step 1](../../../static/img/spm/xcode-step-1.png)
+
+Highlight App, and Select the Package Dependencies tab, and on this page press the + symbol to add a dependency:
+
+![Migrate Step 2](../../../static/img/spm/xcode-step-2.png)
+
+You should see something similar to the below - select Add Local... from the dialog:
+
+![Migrate Step 3](../../../static/img/spm/xcode-step-3.png)
+
+Select CapApp-SPM in this dialog and click Add Package:
+
+![Migrate Step 4](../../../static/img/spm/xcode-step-4.png)
+
+Click Add Package again when this screen shows up: 
+
+![Migrate Step 5](../../../static/img/spm/xcode-step-5.png)
+
+When you are done, you should see a screen like this. At this point you're done and can build and work as normal:
+
+![Migrate Step 6](../../../static/img/spm/xcode-step-6.png)
 
 ### Converting existing plugins to SPM
```

Comment:
The ticket also mentioned
> `Converting existing plugins to SPM`- this section says "more coming soon"; it should probably be filled out

But this section is unchanged, should be updated too